### ISSUE TITLE
After uninstalling, comments stayed hidden

### ISSUE THREAD

**Issue by dlgombert** [estruyf/vscode-hide-comments#10]
Had to reinstall, and set to "show comments" before uninstalling in order to fix.

**Comment by estruyf** [comment]
Correct, unfortunately there is no uninstall event that has or can use the current VSCode instance. 

That is why I provided uninstall steps in the documentation.

Once there is a better way, this will be implemented.
